modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptAngularClientCodegen.java

```diff
@@ -449,13 +455,13 @@ public String toApiName(String name) {
         }
         return initialCaps(name) + serviceSuffix;
     }
-
+    
     @Override
     public String toApiFilename(String name) {
         if (name.length() == 0) {
             return "default.service";
-        }
-        return camelize(removeModelSuffixIfNecessary(name), true) + serviceFileSuffix;
+        }        
+		return this.convertUsingFileNamingConvention(name) + serviceFileSuffix;
```

Comment:
Please use spaces instead of tabs and remove trailing spaces at line 463

Reply:
Please replace all tabs with 4-space in this file. Let me know if you need any help on that.